integration_test/perf_driver.dart

```diff
@@ -0,0 +1,19 @@
+// This integration driver configures output of timeline data
+// and a summary thereof from integration tests. See
+// docs/integration_tests.md for background.
+
+import 'package:flutter_driver/flutter_driver.dart' as driver;
+import 'package:integration_test/integration_test_driver.dart';
+
+Future<void> main() {
+  return integrationDriver(
```

Comment:
```suggestion
Future<void> main() {
  // See cookbook recipe for this sort of driver:
  //   https://docs.flutter.dev/cookbook/testing/integration/profiling#3-save-the-results-to-disk
  return integrationDriver(
```

That way the relevant bit of docs is handy when editing this file in the future (cf https://github.com/zulip/zulip-flutter/pull/317#discussion_r1376979715 ).